lxd/instance/instance_utils.go

```diff
@@ -1049,9 +1050,14 @@ func NextSnapshotName(s *state.State, inst Instance, defaultPattern string) (str
 	return pattern, nil
 }
 
-// temporaryName concatenates the move prefix and instUUID for a temporary instance.
-func temporaryName(instUUID string) string {
-	return fmt.Sprintf("lxd-move-of-%s", instUUID)
+// temporaryName returns the temporary instance name using a stable random generator.
+func temporaryName(instUUID string) (string, error) {
```

Comment:
lets land this commit as a separate PR

Reply:
Moved with https://github.com/canonical/lxd/pull/12568